packages/suite/src/components/suite/modals/TransactionDetail/components/IODetails.tsx

```diff
@@ -454,7 +454,8 @@ export const IODetails = ({ tx }: IODetailsProps) => {
 
     return (
         <Wrapper>
-            <AnalyzeInBlockbookBanner txid={tx.txid} />
+            {/* solana is not supported by blockbook */}
+            {network?.networkType !== 'solana' && <AnalyzeInBlockbookBanner txid={tx.txid} />}
```

Comment:
poor naming, it should be maybe `AnalyzeInExplorerBanner` because `explorer` url from networksConfig is used here, so the banner can stay here for solana too.

Reply:
Thanks, pls check https://github.com/trezor/trezor-suite/pull/9498/commits/82804ed5c9316ca029f2a0e933ec59064c4fd56d where I renamed it as you suggest. 

Let me just point out that the Banner it self says `Analyze in Trezor Blockbook` also for solana and other blockchains which do not use blockbook.